superset/assets/javascripts/SqlLab/components/CopyQueryTabUrl.jsx

```diff
@@ -9,16 +9,14 @@ const propTypes = {
 export default class CopyQueryTabUrl extends React.PureComponent {
   getUrl(callback) {
     const qe = this.props.queryEditor;
-    const params = [];
-    if (qe.dbId) params.push('dbid=' + qe.dbId);
-    if (qe.title) params.push('title=' + encodeURIComponent(qe.title));
-    if (qe.schema) params.push('schema=' + encodeURIComponent(qe.schema));
-    if (qe.autorun) params.push('autorun=' + qe.autorun);
-    if (qe.sql) params.push('sql=' + encodeURIComponent(qe.sql));
-
-    const queryString = params.join('&');
-    const queryLink = window.location.pathname + '?' + queryString;
-    getShortUrl(queryLink, callback);
+    const sharedQuery = {
+      dbId: qe.dbId,
```

Comment:
qe already has those variables as you pull them from it. 
Question was will it make sense to package them into the query object this PR introduces this concept.